tests/e2e/specs/mutation.cy.js

```diff
@@ -199,4 +199,18 @@ describe('Mutations component', () => {
       .get('@errTooltip')
       .should('not.have.css', 'display', 'none')
   })
+  it('should display action buttons in menu card actions component', () => {
```

Comment:
Just a slight clarification
```suggestion
  it('has actions buttons pinned to bottom of form', () => {
```